_tests/browser_tests/spec_suites/shared/shared_the-bureau.js

```diff
@@ -16,4 +16,4 @@ describe( 'Beta The Bureau Page', function() {
     expect( page.missions.count() ).toEqual( 3 );
     expect( page.missions.getText() ).toEqual[ 'Educate', 'Enforce', 'Empower' ];
   } );
-} );
+} );
```

Comment:
I just shared my user settings in the GHE D&D Team repo, everyone should check it out and add their own.